sim/shaman/_earthquake.go

```diff
@@ -0,0 +1,52 @@
+package mage
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+)
+
+func (shaman *Shaman) registerEarthquakeSpell() {
```

Comment:
I wasn't sure if you meant for these to be shown or not but files with an underscore at the start of the name won't be loaded